GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -216,43 +216,43 @@ public IReadOnlyList<ObjectId> SelectedObjectIds
                 }
 
                 lock (_backgroundEvent)
-                lock (_graphModel)
-                {
-                    if (value == null)
+                    lock (_graphModel)
```

Comment:
This formatting change makes the PR harder to review.

Reply:
Sorry, but there is an easy solution
![image](https://user-images.githubusercontent.com/483659/44981494-64190680-af7b-11e8-91c7-c3397b4d9c9f.png)

On the other hand, this might not be following formatting rules. Shouldn't block level statements be nested to highlight block level? (I do agree that previous formatting looks sexier though)